projects/batfish/src/main/java/org/batfish/config/Settings.java

```diff
@@ -372,6 +372,10 @@ public void setTopologyPath(Path path) {
 
   private static final String ARG_ANONYMIZE = "anonymize";
 
+  private static final String ARG_CONTAINERS_LOCATION = "containerslocation";
```

Comment:
Container location should be coming from the Coordinator, not from the command line. It should probably come when the worker registers with the Coordinator:

- Worker: "I am a new worker"
- Coordinator: "Great! We use this storage: <x>"

(or something like that.)